zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -146,6 +156,14 @@ impl<'a> zp_tx::Authorization for PrecomputedAuth<'a> {
     type TransparentAuth = TransparentAuth<'a>;
     type SaplingAuth = sapling_crypto::bundle::Authorized;
     type OrchardAuth = orchard::bundle::Authorized;
+
+    // FIXME: is this correct?
+    #[cfg(zcash_unstable = "nu6")]
+    type OrchardZsaAuth = orchard::bundle::Authorized;
+
```

Comment:
This seems unnecessary if it's going to be the same as the `OrchardAuth` type?

Reply:
I had to duplicate it because is `OrchardZsaAuth` required by `Authorization` trait:

https://github.com/QED-it/librustzcash/blob/zsa1/zcash_primitives/src/transaction/mod.rs#L303